Android/app/src/main/res/layout/activity_login.xml

```diff
@@ -139,16 +139,32 @@
                 android:visibility="gone"
                 app:layout_behavior="@string/appbar_scrolling_view_behavior">
 
+                <android.support.design.widget.TextInputLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:textColorHint="#ffffff">
+
+                    <EditText
+                        android:id="@+id/input_firstname_signup"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:hint="@string/first_name"
+                        android:inputType="text"
+                        android:singleLine="true"
+                        android:textColor="#ffffff"
+                        android:textColorHint="#ffffff" />
+                </android.support.design.widget.TextInputLayout>
+
                 <android.support.design.widget.TextInputLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:textColorHint="#ffffff">
 
                     <EditText
-                        android:id="@+id/input_name_signup"
+                        android:id="@+id/input_lastname_signup"
```

Comment:
This shouldn't be a part of this fix

Reply:
In description it was given to add Last name field during sign up